e2e/fixtures/externalVC.ts

```diff
@@ -0,0 +1,175 @@
+export const validExemplaryExternalVC = {
```

Comment:
Minor name change suggestion. To me, "Exemplary" suggests that it is an ideal or very good credential. Is that what you mean?

Were these credentials copied or generated from somewhere? If so, I suggest putting a link to that spot or code.

```suggestion
export const validExampleExternalVC = {
```

Reply:
I copied it from one of the spruce project. And yeah it is just an example, but working one so I think we can think of it like "very good" :smile: I will rename it